pages/mesosphere/dcos/services/edge-lb/1.6/related-documentation/release-notes/index.md

```diff
@@ -0,0 +1,293 @@
+---
+layout: layout.pug
+navigationTitle:  Release notes
+title: Release notes
+menuWeight: 101
+excerpt: Provides release-specific information for Edge-LB
+enterprise: false
+---
+
+These Edge-LB Release Notes summarize release-specific changes that fix issues or update Edge-LB features for DC/OS&trade; Enterprise clusters.
+
+# DC/OS for Edge-LB Service version 1.6.0 Release Notes
+
+Edge-LB Service version 1.6.0 was released on 11 June 2020.
```

Comment:
NIT: Please update to the actual release date.

Reply:
Updated:

```sh
curl -s --head https://downloads.mesosphere.com/edgelb/v1.6.0/assets/stub-universe-edgelb.json | grep -i last-modified
last-modified: Wed, 10 Jun 2020 22:07:32 GMT
```